controllers/error-controller.js

```diff
@@ -0,0 +1,11 @@
+exports.handlePSQL400s = (error, request, response, next) => {
+    if (error.code === '22P02') {
```

Comment:
We can remove this 404 at this point and instead use the wild card to handle a default 404 in app.js - just leave your 500 psql error handler here